.github/renovate-configs/README.md

```diff
@@ -14,7 +14,7 @@ follows (the double slash is intentional!) :
 ```
 {
   "extends": [
-    "github>GoogleCloudPlatform/kubernetes-engine-samples//.github/renovate-configs:purple.json5",
+    "github>GoogleCloudPlatform/kubernetes-engine-samples//.github/renovate-configs/purple.json5",
```

Comment:
**Question:** Is there a reason that you suspect the `:` (colon) is an issue?
The [docs you shared earlier](https://docs.renovatebot.com/config-presets/#:~:text=GitHub%20with%20preset%20name%20(JSON5)) indicate that we need to use `:` (colon) for JSON 5.

Reply:
NVM, I just found [this example](https://github.com/syself/cluster-api-provider-hetzner/blob/e6128520aa716acb3a2b75c19e3e620b7e3e13c6/.github/renovate.json5#L10).

I'm likely misinterpreting the Renovate docs. 😕

Reply:
Yeah, this is a consequence of renovate's peculiar syntax for the reference - apparently if you're [using a subdirectory path (indicated by `//`), the colon syntax is not supported](https://docs.renovatebot.com/config-presets/#github:~:text=anything%20in%20the%20form%20provider%3Eowner/repo//path/to/file%3Asubsubpreset%20is%20not%20supported.) and throws an error, but the full file path works just fine 🤷🏼